pylib/mozhginfo/setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='pushlog_client',
+    version='0.1',
+    packages=find_packages(),
+
+    install_requires=[
+        'requests>=2.5.1'
```

Comment:
requests 2.6.0 contains a CVE fix. And the latest version has an updated cert bundle. For security reasons, please require the newest released requests package, which is currently 2.8.1.